### ISSUE TITLE
Remove credits from main menu

### ISSUE THREAD

**Issue by dyceron** [randovania/open-samus-returns-rando#187]
For when the spoiler log is implemented

**Comment by Miepee** [comment]
What exactly is the issue by having credits available from title sceen? That you can spoil a game by watching credits?

**Comment by dyceron** [comment]
Yeah pretty much, which is why if it's a race seed, the spoiler isn't added, so the credits don't mention items.

**Comment by Miepee** [comment]
Is there some variable that gets you the last loaded "room"? Would it be possible to add one?
